semantic-conventions/README.md

```diff
@@ -83,6 +83,9 @@ convention that have the tag `network`.
 `<!-- semconv http.server(tag=network, full) -->` will print the constraints and attributes of both `http` and `http.server`
 semantic conventions that have the tag `network`.
 
+`<!-- semconv metric.http.server(metric_table) -->` will print a table of metrics with all metrics prefixed with 
```

Comment:
Is this really depending on the prefix? Probably on the group ID?

Reply:
Good catch, this example should've changed when I made the changes to the PR. 

```
`<!-- semconv metric.http.server.active_requests(metric_table) -->` will print a table describing a single metric 
`http.server.active_requests`. 
```

The group ID is `metric.http.server.active_requests` but the `metric_name` (a new field I added) will be printed to the markdown, which in this case would be `http.server.active_requests`.